### DISCUSSION TITLE
Strange URL Issue

### DISCUSSION THREAD

**Discussion by TheDarthFather** [firefly-iii/firefly-iii#5900] [Firefly III] [answered]
Hello,
I am trying to install the importer into a docker next to my firefly docker instance.
Initialy I setup the FIREFLY_III_URL to my non-docker IP (http://192.168.0.8:8019 ) and then I realised I have to set it up, referencing my docker container IP and port (http://172.17.0.9:8080).
Even if I rebuild the container ... I get this error .. that still points to the host network IP. 
It seems is kept somewhere and I cannot override it ... I feel i've done something stupid and simple to fix ... but I dont realise what is this is it ...
Any hints ?

`[2022-03-15 20:22:12] production.ERROR: Could not connect to Firefly III: Error accessing "http://192.168.0.8:8019/api/v1/about". Status code is 401. Body is: {"message":"Unauthenticated","exception":"AuthenticationException"}  `

**Comment by JC5** [comment, marked as answer]
Browse to /flush, press "start over" or clear your cookies. That should do it!

**Reply by TheDarthFather** [reply]
Indeed :)
